qiskit_nature/problems/sampling/protein_folding/peptide/beads/base_bead.py

```diff
@@ -37,7 +37,7 @@ def __init__(
         Args:
             chain_type: Type of the chain, either "main_chain" or "side_chain".
             main_index: index of the bead on the main chain in a peptide.
-            residue_type: A character representing the type of a residue for the bead. None in
+            residue_type: A character representing the type of a residue for the bead. "" in
```

Comment:
Maybe write `an empty string` instead of `""`? Same elsewhere (presumably..)

Reply:
I corrected in other places and somehow skipped this one. My bad.